doc/source/rllib-dev.rst

```diff
@@ -0,0 +1,134 @@
+RLlib Developer Guide
+=====================
+
+.. note::
+
+    If you want to apply existing RLlib algorithms, first check out the `user docs <http://ray.readthedocs.io/en/latest/rllib.html>`__.
```

Comment:
This guide will take you through steps for implementing a new algorithm in RLlib. To apply existing algorithms already implemented in RLlib, please see the user docs.

Reply:
Done